cucumber/authenticators_jwt/features/authn_jwt.feature

```diff
@@ -130,3 +130,58 @@ Feature: JWT Authenticator - JWKs Basic sanity
     """
     CONJ00100E Annotation, 'custom-claim', is empty
     """
+
+  Scenario: Ignore invalid annotations
+    Given I have a "variable" resource called "test-variable"
+    Given I extend the policy with:
+    """
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw: invalid
+        authn-jwt: invalid
+        authn-jwt/raw/sub/sub: invalid
+        authn-jwt/raw2/sub: invalid
+        authn-jwt/raw/sub: valid
+        authn-jwt/raw/namespace-id: valid
+        authn-jwt/raw/project-path: valid
+    """
+    And I permit host "myapp" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
```

Comment:
Could you put this lined right after the first line? making them together will make more sense..